examples/webgl_instancing_dynamic.html

```diff
@@ -93,49 +154,68 @@
 
 			function animate() {
 
-				render();
+				timer.update();
 
-				stats.update();
+				const time = timer.getElapsed();
 
-			}
+				TWEEN.update();
+
+				// animate camera
+
+				camera.position.x = Math.sin( time / 4 ) * 10;
+				camera.position.z = Math.cos( time / 4 ) * 10;
+				camera.position.y = 8 + Math.cos( time / 2 ) * 2;
+
+				cameraTarget.x = Math.sin( time / 4 ) * - 8;
+				cameraTarget.z = Math.cos( time / 2 ) * - 8;
+
+				camera.lookAt( cameraTarget );
```

Comment:
Can you please stop the unnecessary spinning? Keep the camera still. The demo will look very nice without it.

Reply:
IMO, the animation makes the demo more exciting. Besides, the previous version of the demo also had an animation. Insteadof rotating the entire instanced mesh, the camera is now animated. I would prefer to keep the status quo. Not just because it looks better but also to honor the style of the example's reference from oosmoxiecode.